lib/managers/terms-of-service.js

```diff
@@ -0,0 +1,251 @@
+	var params = {
+		body: {
+			status: termsOfServicesStatus,
+			text: termsOfServicesText
+		}
+	};
+
+	var apiPath = urlPath(BASE_PATH, termsOfServicesID);
+	return this.client.wrapWithDefaultHandler(this.client.put)(apiPath, params, callback);
+};
+
+/**
+ * Gets a specific custom terms of services with specified ID
+ *
+ * API Endpoint: '/terms_of_services/:termsOfServicesID'
+ * Method: GET
+ *
+ * @param {string} termsOfServicesID - The id of the custom terms of services to retrieve
+ * @param {Object} [options] - Additional options. Can be left null in most cases.
+ * @param {string} [options.fields] - Comma-separated list of fields to return on the collaboration objects
+ * @param {Function} [callback] - Passed the terms of services information with specified ID if successful, error otherwise
+ * @returns {Promise<Object>} A promise resolving to the terms of services object
+ */
+TermsOfService.prototype.getTermsOfServicesByID = function(termsOfServicesID, options, callback) {
+	var params = {
+		qs: options
+	};
+
+	var apiPath = urlPath(BASE_PATH, termsOfServicesID);
+	return this.client.wrapWithDefaultHandler(this.client.get)(apiPath, params, callback);
+};
+
+/**
+ * Gets custom terms of services for the user's enterprise
+ *
+ * API Endpoint: '/terms_of_services'
+ * Method: GET
+ *
+ * @param {Object} [options] - Additional options. Can be left null in most cases.
+ * @param {string} [options.tosType] - Optional, indicates whether the terms of service is set for external or managed under enterprise
+ * @param {string} [options.fields] - Comma-separated list of fields to return on the collaboration objects
+ * @param {Function} [callback] - Passed the terms of services information if successful, error otherwise
+ * @returns {Promise<Object>} A promise resolving to the terms of services object
+ */
+TermsOfService.prototype.getTermsOfServices = function(options, callback) {
+	var params = {
+		qs: options
+	};
+
+	var apiPath = urlPath(BASE_PATH);
+	return this.client.wrapWithDefaultHandler(this.client.get)(apiPath, params, callback);
+};
+
+/**
+ * Accepts/rejects custom terms of services for the user
+ *
+ * API Endpoint: '/terms_of_service_user_statuses'
+ * Method: POST
+ *
+ * @param {string} termsOfServicesID - Terms of services ID to retrieve user statuses on
+ * @param {bool} isAccepted - Determines wehether the terms of services has been accepted or rejected
+ * @param {Object} [options] - Additional options. Can be left null in most cases.
+ * @param {string} [options.termsOfServiceUserID] - Optional, user id to retrieve terms of service status on, default is current user
+ * @param {Function} [callback] - Passed the terms of service user status information if successful, error otherwise
+ * @returns {Promise<Object>} A promise resolving to the terms of service user status
+ */
+TermsOfService.prototype.createTermsOfServiceUserStatuses = function(termsOfServicesID, isAccepted, options, callback) {
+	var tos = {
```

Comment:
Consider inlining this object definition into the body below so it's easier to see the full shape of the payload